wlc/test_wlc.py

```diff
@@ -135,6 +135,30 @@ def test_repr(self):
         self.assertIn("'slug': 'hello'", repr(obj))
         self.assertIn("'slug': 'hello'", str(obj))
 
+    def test_create_project(self):
+        resp = Weblate().create_project(
+            "Hello", "hello", "http://example.com/", "Malayalam", "ml"
+        )
+        self.assertIn("'name': 'Hello', 'slug': 'hello'", str(resp))
```

Comment:
Why aren't we parsing the response and testing specific attribute values one by one?

Reply:
I did think that we could add some more checks there. The existing tests were minimal in nature, so I was following the pattern.
I'll update the tests to be more thorough. 😅